docs/howto/openstack/nova/config-drive.md

```diff
@@ -20,35 +20,64 @@ configuration source.
 ## Store metadata on a configuration drive
 
 A **configuration drive** (config drive) is a read-only virtual drive
-that is attached to a server during boot. The server can then
+that gets attached to a server during boot. The server can then
 mount the drive and read files from it. Configuration drives are used
 as a data source for
 [cloud-init](https://cloudinit.readthedocs.io/en/latest/).
 
 
-## Enable the configuration drive on server creation (`openstack` CLI)
+## Enable the configuration drive on server creation
 
-To enable the configuration drive, you need to pass the parameter
-`--use-config-drive` to the `openstack server create` command.
+Follow our [How-To guide](new-server.md) to create your new cloud server
```

Comment:
It's perfectly fine to cross-reference things like that, but if you do, then please also add a back-reference. In other words, insert a link in `new-server.md`, right above or below the `shot-08.png` screenshot, pointing readers to *this* guide in case they want to use config-drive.